docs/tuning-apps/networking/tcp.rst

```diff
@@ -247,20 +247,14 @@ not use specific IP networks |mdash| or not use any IP networks at all
 .. warning:: If you use the ``btl_tcp_if_include`` and
              ``btl_tcp_if_exclude`` MCA parameters to shape the
              behavior of the TCP BTL for MPI communications, you may
-             also need/want to investigate the corresponding MCA
-             parameters ``oob_tcp_if_include`` and
-             ``oob_tcp_if_exclude``, which are used to shape non-MPI
-             TCP-based communication (e.g., communications setup and
-             coordination during ``MPI_INIT`` and ``MPI_FINALIZE``).
-
-.. error:: TODO do corresponding OOB TCP params still exist in PMIx?
-
-Note that Open MPI will still use TCP for control messages, such as
-data between ``mpirun`` and the MPI processes, rendezvous information
-during ``MPI_INIT``, etc.  To disable TCP altogether, you also need to
-disable the ``tcp`` component from the OOB framework.
-
-.. error:: TODO Is this possible in PMIx?  I doubt it...?
+             also need/want to investigate the corresponding PRRTE
+             parameters that control use of network interfaces by the runtime
+             (e.g., communications setup and coordination during
+             ``MPI_INIT`` and ``MPI_FINALIZE``) using the `prte_info command <https://docs.prrte.org/en/latest/man/man1/prte_info.1.html>`_.
```

Comment:
I just tested this to be sure, but in the v5.0.x tarball (and therefore likely the main/v6.0.x tarballs), we include the prte_info.1 HTML man page.  Is there a way we can link to that here, rather than linking to the one in docs.prrte.org?

Reply:
Good question - let me look. I _think_ the prte_info.1 may be something in OMPI and not in PRRTE. If so, I'll steal it for upstream.

Reply:
Well, everything checks out - PRRTE is generating a prte_info man page. From what I can see from a quick scan, your build system doesn't pick that up and install it in your prefix path. So it's available, but not in your install path.